tests/test_hydra.py

```diff
@@ -72,6 +72,20 @@ def test_run_dir() -> None:
     get_run_output(["tests/test_apps/run_dir_test/my_app.py"])
 
 
+@pytest.mark.parametrize("rundir", ["\\${now:%Y%m%d_%H%M%S_%f}"])
+def test_run_dir_microseconds(tmpdir: Path, rundir: str) -> None:
+    rundir = str(tmpdir / rundir)
+    cmd = [
+        "tests/test_apps/run_dir_test/my_app_microseconds.py",
+        f"hydra.run.dir={rundir}",
+    ]
```

Comment:
You can use integration_test() instead of this decicated app.
Take a look at how it's being used in other places.

Reply:
It seems to expect something exact, but with microseconds I'm unsure how timestamp will go?

Reply:
You could print something like:
'%f' not in os.getcwd() and ensures that it's "True".

Reply:
Then I might try assert_regex_match, then it seems to only have use cases with dedicated programs

Reply:
```
You could print something like:
'%f' not in os.getcwd() and ensures that it's "True".
```

OK I see